dev-util/sysprof/sysprof-45_rc.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson systemd xdg
+
+DESCRIPTION="System-wide Linux Profiler"
+HOMEPAGE="http://sysprof.com/"
+
+LICENSE="GPL-3+ GPL-2+"
+API_VERSION="4"
+SLOT="0/${API_VERSION}"
```

Comment:
In gnome-builder, we have a requirement on `sysprof-6`:
diff```
-libsysprof_dep = dependency('sysprof-4', version: sysprof_req)
-libsysprof_ui_dep = dependency('sysprof-ui-5', version: sysprof_req)
+libsysprof_dep = dependency('sysprof-6', version: sysprof_req)
```

And sysprof provides only one version of the shared object (here's the diff for sysprof-3.48 to the version in this PR):
```diff
-libsysprof_api_version = 4
-libsysprof_ui_api_version = 5
+# All libraries share an ABI revision as they are expected
+# to be updated as a set. However, we keep libsysprof-capture
+# at an older version since it's used as a static library
+# from various platform tooling
+soname_major_version = 6
+libsysprof_capture_api_version = 4
+libsysprof_api_version = soname_major_version
```

```
framework /var/tmp/portage/dev-util/sysprof-3.48.0/image # find . | grep lib64
./usr/lib64
./usr/lib64/pkgconfig
./usr/lib64/pkgconfig/sysprof-ui-5.pc
./usr/lib64/pkgconfig/sysprof-4.pc
./usr/lib64/libsysprof-ui-5.so
./usr/lib64/libsysprof-speedtrack-4.so
./usr/lib64/libsysprof-memory-4.so
./usr/lib64/libsysprof-4.so
```

```
framework /var/tmp/portage/dev-util/sysprof-45_rc/image # find . | grep lib64
./usr/lib64
./usr/lib64/libsysprof-6.so
./usr/lib64/libsysprof-6.so.6
./usr/lib64/pkgconfig
./usr/lib64/pkgconfig/sysprof-6.pc
./usr/lib64/libsysprof-tracer-6.so
./usr/lib64/libsysprof-speedtrack-6.so
./usr/lib64/libsysprof-memory-6.so
./usr/lib64/libsysprof-6.so.6.0.0
```

I wonder if we need to split this out into a separate package and slot it? :(